### PR TITLE
added a toggle on/off switch

### PR DESCRIPTION
**Pull request by devhl-labs**
This will add a toggle switch to the popup.  The switch enables and disables the dark theme.  Set the switch, then reload the GitHub page.

### CODE REVIEW

**Comment by EmaSuriano** [comment]
Hoping for this one! 👏

File: src/pages/popup.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+		console.log('isEnabled is false');
+	}
+	console.log(isEnabled);
+});
+
+
+
+document.getElementById('isEnabled').addEventListener('click', (event) => {
+	console.log('isEnabled was clicked');
+	(event.target.checked) ? chrome.storage.sync.set({'isEnabled' : "true"}) : chrome.storage.sync.set({'isEnabled' : "false"});
```
Comment (SimplyBallistic) [inline src/pages/popup.js:20]:
```suggestion
	chrome.storage.sync.set({'isEnabled' : (!!event.target.checked).toString()});
```
casting to boolean (!!) and then to string. However, we know event.target.checked is always a boolean AFAIK so just removing the cast can work too.

File: src/pages/popup.html [hunk 2]
```diff
@@ -10,6 +10,72 @@
...
+			  display: inline-block;
+			  width: 60px;
+			  height: 34px;
+			}
+
+			/* Hide default HTML checkbox */
+			.switch input {
+			  opacity: 0;
+			  width: 0;
+			  height: 0;
```
Comment (SimplyBallistic) [inline src/pages/popup.html:25]:
```suggestion
			display: none;
```
Does this work here?

**Comment by SimplyBallistic** [comment]
Fixes #159

**Comment by devhl-labs** [comment]
No, it does not update instant, you have to reload the tab.  It only targets Chrome.  I wasn't sure how to do it so it supports everything this extension supports.

**Comment by SimplyBallistic** [comment]
> 
> 
> No, it does not update instant, you have to reload the tab. It only targets Chrome. I wasn't sure how to do it so it supports everything this extension supports.

If you update your fork to the latest commit of this repo, you should get the new changes that let you easily support all the browsers. I believe its done with [webextension-polyfill](https://github.com/mozilla/webextension-polyfill), which is very similar to the chrome API. With minimal changes, you'll be able to support them all!

However, it doesn't seem the author is interested with checking up on PRs such as this one, hopefully it gets some attention in the future 🤞

**Comment by poychang** [comment]
@SimplyBallistic I am interested this PR. But I think this needs more works to fit the request. I still thinking about that. And at the mean time, I have #123 more critical issue to figure out. Thanks for all of you to support and interested this extension.

**Comment by devhl-labs** [comment]
If I have time I'll try to improve the pr.

**Comment by igorrynkovoy** [comment]
The only reason not to use this extension - there is no ON/OFF option. Hope this will be released soon.

**Pull request closed without merging**